src/components/Assets/AssetTable.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { handleTableDataStringLength, formatDataTitles } from "./helpers";
+
+const AssetTable = ({ fields, data }) => {
+  const tableHeaders = fields.map(field =>
+    formatDataTitles(Object.keys(field))
+  );
+  const fieldIds = fields.map(field => {
+    return Object.values(field)[0];
+  });
+  // TODO add responsive font size (or rely on side-scroll?)
```

Comment:
yeah, I think side scroll is going to have to be the solution unfortunately. Can you make sure that works as expected on tablet and that its not a total flop.

Reply:
Cool, it looks good on a tablet. I think it should work out for users.